Microsoft.Toolkit.Uwp.UI.Controls.Input/TokenizingTextBox/TokenizingTextBox.Properties.cs

```diff
@@ -157,6 +157,46 @@ private static void TextPropertyChanged(DependencyObject d, DependencyPropertyCh
             typeof(TokenizingTextBox),
             new PropertyMetadata(false));
 
+        /// <summary>
+        /// Identifies the <see cref="MaxTokens"/> property.
+        /// </summary>
+        public static readonly DependencyProperty MaxTokensProperty = DependencyProperty.Register(
+            nameof(MaxTokens),
+            typeof(int),
+            typeof(TokenizingTextBox),
+            new PropertyMetadata(DependencyProperty.UnsetValue, OnMaxTokensChanged));
```

Comment:
Nope, needs to be `null`! `UnsetValue` is the special value used by the framework. If you do this, then you can never tell when the developer set it properly. This is explicitly called out [in the docs](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/custom-dependency-properties#default-value).

Reply:
> Do not register with a default value of UnsetValue. If you do, it will confuse property consumers and will have unintended consequences within the property system.

Good to know! I'll use `null`